lib/parsers/pdf_parser.rb

```diff
@@ -6,7 +6,7 @@ class FormatParser::PDFParser
   #
   # There are however exceptions, which are left out for now.
   #
-  PDF_MARKER = /%PDF-1\.[0-8]{1}/
+  PDF_MARKER = /%PDF-[1-2]\.[0-8]{1}/
```

Comment:
Wouldn't this be enough, for now? Don't want to go in the performance argument, but it just reads a bit simpler.

```suggestion
  PDF_MARKER = /%PDF-[12]\.[0-8]{1}/
```

Reply:
Any digit from 1 to 2, potentially from 1 to a higher digit later on. I think it works fine as is.

Reply:
By the way, I could have used `\d` to allow any digit, but I thought it's better to support these specific versions only, as any newer version may introduce breaking changes that we may want to review before automatically parsing it... What do you think?

Reply:
Any digit from 1 to 2 indeed; an interval doesn't really make sense if we are talking about the ends of it only 😊 

I agree with you that `\d` is not ideal in this situation.

Reply:
Done. I've removed the dash.